src/app/components/flyout/flyout-guidelines.component.html

```diff
@@ -0,0 +1,273 @@
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-bg-active-2.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-bg-active-3.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+		<div class="sky-subsection-heading">Independent scrolling</div>
+			<p>The flyout and its background page scroll independently. When users touch, hover, or focus on the background page, only the background content scrolls. Similarly, when users touch, hover, or focus on the flyout, only the flyout scrolls. This allows users to adjust the content in the flyout without losing context from the page.</p>
+			<stache-row>
+    		<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-scroll-1.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-scroll-2.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-scroll-3.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+		<div class="sky-subsection-heading">Drag to resize</div>
+			<p>Users can select the flyout's left border and drag it to adjust the size. The size can range from 320px to 20px less than the full screen size. If users expand the flyout to the full screen size, it slides 20px to the right to keep a portion of the background page in view. The default size is 50 percent of the screen or 320px, whichever is larger. When users resize the panel, the system remembers their preferred size globally.</p>
+			<stache-row>
+    		<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-resize-1.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-resize-2.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenMedium="4">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-resize-3.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+		<div class="sky-subsection-heading">Help panel</div>
+			<p>The flyout covers the omnibar, but the help panel remains visible in the upper right corner of the screen. The help panel should recognize that the flyout is open and provide appropriate guidance based on the context.</p>
```

Comment:
We reference the omnibar here, but it is for internal use only. Should we replace "omnibar" with "top-level navigation"?